### PR TITLE
[#63] Implement weight feature

### PR DESCRIPTION
**Pull request by hangpark**
See #63.

Default weight parameter is set to be 32.

### CODE REVIEW

File: app/src/main/java/kr/ac/kaist/team888/util/FeatureController.java [hunk 1]
```diff
@@ -19,7 +19,8 @@ public int compare(WeakReference o1, WeakReference o2) {
     }
   };
 
-  private float curve = 0.0f;
+  private double curve = 0.0f;
```
Comment (xeoh) [inline app/src/main/java/kr/ac/kaist/team888/util/FeatureController.java]: it's double. ```0.0```

File: app/src/main/java/kr/ac/kaist/team888/util/FeatureController.java [hunk 2]
```diff
@@ -19,7 +19,8 @@ public int compare(WeakReference o1, WeakReference o2) {
...
+  private double weight = 0.5f;
```
Comment (xeoh) [inline app/src/main/java/kr/ac/kaist/team888/util/FeatureController.java]: ```0.5```

**Review by xeoh — CHANGES_REQUESTED** [review]

**Review by bimak-server — APPROVED** [review]
👍

**Review by xeoh — APPROVED** [review]
👍

**Pull request merged**